requirements.txt

```diff
@@ -12,6 +12,6 @@ scandir>=1.2
 
 # For testing
 pytest>=2.9
-pytest-cov>=2.2
+pytest-cov==2.6.0
```

Comment:
A new version of pytest-cov had been released two days ago. Just to allow the tests to run I've added this requirement. Itamar, what would you suggest doing?

Reply:
what's wrong with the new version?

the `>=2.2` part shouldn't change - it's still ok to use version 2.2.

it's best practice to avoid pinning an exact version in requirements.txt, so if there is an issue with 2.6.2 for example, you should add `and also <2.6.2` (don't remember the exact syntax for multiple conditions)